source/MyProcessing.mc

```diff
@@ -433,6 +433,14 @@ class MyProcessing {
           $.oMySportsTrackLive.iCounter++;
         }
         
+        //FlySafe
+        if ($.oMyActivity != null && $.oMySettings.iFlySafeLivetrackFrequencySeconds != 0) {
+          if ($.oMyFlySafeLivetrack.iCounter % $.oMySettings.iFlySafeLivetrackFrequencySeconds == 0) {
+            $.oMyFlySafeLivetrack.updatePosition(_oInfo, oLocation, self.fAltitude);
```

Comment:
Just to make sure I understand, there's no concept of starting and stopping a session/flight in FlySafe? Just sending position info is enough?

Reply:
This is correct, no session required. I was surprised as well but it made the whole thing a lot easier so I'm not complaining :D